repository/SmalltalkCI-Core.package/SCIGemStoneServerConfigSpec.class/properties.json

```diff
@@ -0,0 +1,17 @@
+{
+	"category" : "SmalltalkCI-Core",
```

Comment:
avoid what? it is a common spec so the class needs to be there ... you don't want a class with GemStone in the name? I don't care that much about the name, perhaps you can come up with a better name" Noone is forced to use the class ... not sure what the problem is?